docker/php/Dockerfile

```diff
@@ -34,6 +34,7 @@ RUN apt-get install -y nodejs \
 # Install iconv, mcryot, gd
 RUN apt-get install -y \
         cron \
+        telnet \
```

Comment:
telnet  ??? I would prefer ssh !

Reply:
I do use it for test the port connection.